js/parts-more/PackedBubbleSeries.js

```diff
@@ -422,87 +1086,243 @@ seriesType(
+                    zExtremes[1],
                     minSize,
                     maxSize,
                     value
                 );
-
-                if (value === 0) {
+                if (radius === 0) {
                     radius = null;
                 }
-
                 allDataPoints[i][2] = radius;
                 radii.push(radius);
             });
-
             this.radii = radii;
         },
+        // Draggable mode:
+        /**
+         * Redraw halo on mousemove during the drag&drop action.
+         *
+         * @param {Highcharts.Point} point The point that should show halo.
+         */
+        redrawHalo: function (point) {
+            if (point && this.halo) {
+                this.halo.attr({
+                    d: point.haloPath(
+                        this.options.states.hover.halo.size
+                    )
+                });
+            }
+        },
+        /**
+         * Mouse down action, initializing drag&drop mode.
+         *
+         * @param {global.Event} event Browser event, before normalization.
+         * @param {Highcharts.Point} point The point that event occured.
+         */
+        onMouseDown: function (point, event) {
+            var normalizedEvent = this.chart.pointer.normalize(event);
 
-        alignDataLabel: H.Series.prototype.alignDataLabel
-    }
-);
+            point.fixedPosition = {
+                chartX: normalizedEvent.chartX,
+                chartY: normalizedEvent.chartY,
+                plotX: point.plotX,
+                plotY: point.plotY
+            };
+
+            point.inDragMode = true;
+        },
+        /**
+         * Mouse move action during drag&drop.
+         *
+         * @param {global.Event} event Browser event, before normalization.
+         * @param {Highcharts.Point} point The point that event occured.
+         */
+        onMouseMove: function (point, event) {
+            if (point.fixedPosition && point.inDragMode) {
+                var series = this,
+                    chart = series.chart,
+                    normalizedEvent = chart.pointer.normalize(event),
+                    diffX = point.fixedPosition.chartX - normalizedEvent.chartX,
+                    diffY = point.fixedPosition.chartY - normalizedEvent.chartY,
+                    newPlotX,
+                    newPlotY;
+
+                // At least 5px to apply change (avoids simple click):
+                if (Math.abs(diffX) > 5 || Math.abs(diffY) > 5) {
+                    newPlotX = point.fixedPosition.plotX - diffX;
+                    newPlotY = point.fixedPosition.plotY - diffY;
+
+                    if (chart.isInsidePlot(newPlotX, newPlotY)) {
+                        point.plotX = newPlotX;
+                        point.plotY = newPlotY;
 
-// When one series is modified, the others need to be recomputed
-H.addEvent(H.seriesTypes.packedbubble, 'updatedData', function () {
-    var self = this;
+                        series.redrawHalo();
```

Comment:
This line does nothing - should be `series.redrawHalo(point)`

Reply:
Good catch! It looks that the whole method is not necessary in that case. I have removed it already without any changes in code.